### DISCUSSION TITLE
Actions to take one or multiple windows and move them

### DISCUSSION THREAD

**Discussion by babariviere** [YaLTeR/niri#812] [Q&A] [answered]
Hello, this is a feature that I quite like in paperwm. It allows you to take one or multiple windows and either move all of them at once, or you can dispose them how you like.

Here is a demo: https://private-user-images.githubusercontent.com/30424662/318035381-f736adea-d5ba-4c9d-aca0-2f63322c08cb.mp4?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4ODE3NzYsIm5iZiI6MTczMTg4MTQ3NiwicGF0aCI6Ii8zMDQyNDY2Mi8zMTgwMzUzODEtZjczNmFkZWEtZDViYS00YzlkLWFjYTAtMmY2MzMyMmMwOGNiLm1wND9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTE3VDIyMTExNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY4YTQ4NjQ3NjQ2ZmRlMzllN2MwNGFmYzQ4OGZhZTIwYTVhNTM1MmE1ZDIwMmVkZGNhZjY3NjM1MTVjOTZjNDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.-bHpTn56oN9qMkq1nFeR9tnA0d048ghngQYNXwtSbFQ

Is there something like this in niri?

**Comment by YaLTeR** [comment]
Nothing like this in niri, though you can emulate it by consuming all windows into one column, then moving the column, then expelling them all back out.

**Reply by babariviere** [reply]
Is there any plan to add it? Or do you think it's useless?

**Reply by YaLTeR** [reply, marked as answer]
No particular plan. I haven't used it much myself. There are also multiple designs possible: like in paper, or maybe with some window selection.

**Reply by babariviere** [reply]
Ok thanks! Will live without it for now haha.

**Reply by YaLTeR** [reply]
Btw what use case do you have for this? Like, when is it convenient?

**Reply by babariviere** [reply]
I am just used to it to reorganize a lot of windows, just find it easier to re-spread my windows correctly when I have a lot of terminals and browser windows.

I don't think it should be implemented if you don't see an use-case. This is just a nice to have visual tool.
